.github/workflows/update-on-new-ipfs-tag.yml

```diff
@@ -13,18 +14,19 @@ jobs:
       - name: Find latest go-ipfs tag
         id: latest_ipfs
         uses: ./.github/actions/latest-ipfs-tag
-      - name: Update http-api-docs
+      - name: Update docs
         id: update
-        uses: ./.github/actions/update-on-new-ipfs-tag
+        uses: ./.github/actions/update-with-latest-versions
         with:
           latest_ipfs_tag: ${{ steps.latest_ipfs.outputs.latest_tag }}
-      - name: pull-request  # don't create a pr if there was no new ipfs tag
-        uses: repo-sync/pull-request@65194d8015be7624d231796ddee1cd52a5023cb3 #v2.16
-        if: ${{ steps.update.outputs.updated_branch }}
+      - name: pull-request
+        uses: repo-sync/pull-request@65785d95a5a466e46a9d0708933a3bd51bbf9dde #v2.6.2
```

Comment:
I started seeing a workfspace error (https://github.com/laurentsenta/ipfs-docs/runs/6756806053?check_suite_focus=true) in my github action, had to update. I think the previous version was a typo (2.6.1, not 16)